pages/projects/osg-software.md

```diff
@@ -24,25 +24,58 @@ software systems that comprise the OSG fabric of production services.
+        supporting both concurrently, and allowed for sites to become more familiar with the new technologies.
+
+    -   **Release of OSG 3.6** OSG 3.6 is the first version of the OSG Software Stack without any Globus
+        dependencies.
+        Triggered by the 2017 announcement that Globus was ending support for their open-source Globus Toolkit, the
+        release of OSG 3.6 is the culmination of years of planning and development, and includes versions of HTCondor,
+        HTCondor-CE, and XRootD that rely on token-based authentication instead of X.509 proxy authentication.
+        In addition to the changes in software, the team made a number of release policy changes to ease the transition
+        to token-authentication and to reflect the increasingly agile operational models being explored by the LHC.
+
 -   **Container images:** the OSG Software and Release team has supported the LHC community’s exploration into DevOps
     deployment models by distributing select production services as container images.
-    In particular, the team developed and maintains the [caching](caching.md) container image used by multiple LHC experiments.
-    The team follows a strategy of providing a common base container across all communities, plus community-specific refinements.
-    The team currently supports three communities, U.S. ATLAS, U.S. CMS, and the general OSG community, with [caching](caching.md) container images.
+    In particular, the team developed and maintains the [caching](caching.md) container image used by multiple LHC
+    experiments.
+    The team follows a strategy of providing a common base container across all communities, plus community-specific
+    refinements for U.S. ATLAS and U.S. CMS.
+
+-   **Enterprise Linux 8 support:** the OSG Software Stack is built upon the Enterprise Linux (EL) operating system
+    substrate and new operating system versions requires rebuilding and verification of the entire software stack.
+    EL 8 included a major version upgrade of Python from 2 to 3, resulting in major development efforts by the team and
+    coordination of external software providers.
 
 -   **Streamlining the release process:** to get new software into the hands of site administrators more quickly, the
     team has implemented rolling-style releases along with a policy to make release candidates available more quickly.
+
+-   **Retirement of OSG 3.4:** Retirement of older software is the other side of the coin of maintaining the OSG
+    Software Stack that gives our community timelines to transition to new technologies while ensuring
+    production-readiness of the distributed computing infrastructure. In particular, the retirement of OSG 3.4 meant the
+    end of support for software such as edg-mkgridmap, GUMS, BeStMan, and VOMS Admin Server packages as well as the
+    end-of-life of RedHat Enterprise Linux 6 based operating systems.
```

Comment:
I would add, at the bottom, some variant of the usual IRIS-HEP report text about routine software releases. Treat it like an annual report and throw out some numbers for the past year (you get to define “past year”) — releases, software updates, etc. Whatever you usually include.